README.md

```diff
@@ -1,3 +1,11 @@
+# Nice to meet you
```

Comment:
Nice work!
Can you please move your lines under the link to the discord?

Reply:
Everything seems to be fixed.